cloud_provider/aws/aws_minion_manager.py

```diff
@@ -685,7 +688,7 @@ def minion_manager_work(self):
                         self.update_scaling_group(asg_meta, new_bid_info)
                         continue
 
-                    # Update ASGs iff new bid is different from current bid.
+                    # Update ASGs if new bid is different from current bid.
```

Comment:
The `iff` here was intentional. `iff` was for "if and only if"

Reply:
got it, I reverted the change